CODE_OF_CONDUCT.md

```diff
@@ -20,30 +93,295 @@ Examples of unacceptable behavior by participants include:
+
+Possible private responses to an incident include:
+
+* Nothing, if the behavior was determined to not be a Code of Conduct violation
+* A verbal or emailed warning
+* A final warning
+* Temporarily removing the reported person from the online community
+* Permanently removing the reported person from the online community
+* Publishing an account of the incident
+
+### Work group vote {#work-group-vote}
+
+Decisions on the behavioral modification plans and consequences will be determined by a majority vote of the BIDS Code of Conduct work group.
+
+### Administrators/moderators approval {#administrators/moderators-approval}
+
+Once the work group has approved the behavioral modification plans and consequences, they will communicate the recommended response to the BIDS maintainers. The work group should not state who reported this incident. They should attempt to anonymize any identifying information from the report.
+
+The BIDS maintainers are required to respond back with whether they accept the recommended response to the report. If they disagree with the recommended response, they should provide a detailed response or additional context as to why they disagree. BIDS maintainers are encouraged to respond within a week.
+
+In cases where the BIDS maintainers disagree on the suggested resolution for a report, the BIDS Code of Conduct work group may choose to notify the BIDS steering group.
+
+### Follow up with the reported person {#follow-up-with-the-reported-person}
+
+The BIDS Code of Conduct work group will work with online community administrators/moderators to draft a response to the reported person. The email should contain:
+
+* A description of the person's behavior in neutral language
+* The negative impact of that behavior
+* A concrete behavioral modification plan
+* Any consequences of their behavior
+
+The work group should not state who reported this incident. They should attempt to anonymize any identifying information from the report. The reported person should be discouraged from contacting the reporter to discuss the report. If they wish to apologize to the reporter, the work group can accept the apology on behalf of the reporter.
+
+### Decide further responses {#decide-further-responses}
+
+If the reported person provides additional context, the BIDS Code of Conduct work group may need to re-evaluate the behavioral modification plan and consequences.
+
+### Follow up with the reporter {#follow-up-with-the-reporter}
+
+A person who makes a report should receive a follow-up email stating what action was taken in response to the report. If the work group decided no response was needed, they should provide an email explaining why it was not a Code of Conduct violation. Reports that are not made in good faith (such as "reverse sexism" or "reverse racism") may receive no response.
```

Comment:
```suggestion
A person who makes a report should receive a follow-up email stating what action was taken in response to the report. If the work group decided no response was needed, they should provide an email explaining why it was not a Code of Conduct violation. Reports that are determined not to have been made in good faith may receive no response.
```

I really don't think it's a good idea to pick two particular positions to represent bad faith reports. These phrases have particular meanings in American political discourse, but that meaning may not be shared by people of all international backgrounds or people who have not had their ear to the ground in the last decade or so.

I think stating that the group has discretion to determine bad faith or disparate impact ("Impact" section above) is sufficient to cover these cases without inviting debate over what these terms mean, which debate could itself turn into CoC violations.